.credo.exs

```diff
@@ -21,7 +21,12 @@
         # You can give explicit globs or simply directories.
         # In the latter case `**/*.{ex,exs}` will be used.
         #
-        excluded: [~r"/_build/", ~r"/deps/", ~r"/node_modules/"]
+        excluded: [
+          ~r"/_build/",
+          ~r"/deps/",
+          ~r"/node_modules/",
+          ~r"/lib/dotcom_web/components/prototype.ex"
```

Comment:
Eat it, Credo!

(But actually, I'm not sure about this - we should try to keep our code clean, even if we're putting unfinished features behind basic-authed preview pages)